### ISSUE TITLE
Android release build processes sourcemaps multiple times

### ISSUE THREAD

**Issue by fabriziomoscon** [getsentry/react-native-sentry#245]
OS:
- [ ]  Windows
- [x]  MacOS
- [ ]  Linux

_Platform:_
- [ ]  iOS
- [x]  Android

```
node -v && npm -v && npm ls --prod --depth=0
v8.4.0
5.4.2
```

```
import {
  Sentry,
  SentryLog,
// $FlowFixMe
} from 'react-native-sentry'
Sentry.config(Config.SENTRY_DSN, {
  logLevel: __DEV__ === true ? SentryLog.Debug: SentryLog.None,
}).install()
```

---
I am using `react-native-sentry` `0.24.2` and on Android my release build process generates and uploads sourcemap files 4 times. I am using flavors. Is there any configuration flag or patch I can use to make the build process sourcemaps only once?

```gradle
buildTypes {
        debug {
            minifyEnabled false
            debuggable true
            applicationIdSuffix ".debug"
            versionNameSuffix "-debug"
            proguardFiles getDefaultProguardFile("proguard-android.txt"), "proguard-rules.pro"
        }
        release {
            minifyEnabled enableProguardInReleaseBuilds
            debuggable false
            proguardFiles getDefaultProguardFile("proguard-android.txt"), "proguard-rules.pro"
            signingConfig signingConfigs.release
        }
    }
    productFlavors {
        local {}
        dev {}
        staging {}
        production {}
    }
```

`./gradlew assembleStagingRelease` generates the following message 4 times. the build time went from 2 minutes to 10.

```
Processing react-native sourcemaps for Sentry upload.
> Analyzing 2 sources
> Rewriting sources
> Adding source map references
Uploading sourcemaps for release <MY_PACKAGE_NAME>.staging.debug-1.1-debug distribution 2
> Uploading source maps for release <MY_PACKAGE_NAME>.staging.debug-1.1-debug

Source Map Upload Report
  Minified Scripts
    ~/index.android.bundle (sourcemap at index.android.bundle.map)
  Source Maps
    ~/index.android.bundle.map
Processing react-native sourcemaps for Sentry upload.
> Analyzing 2 sources
> Rewriting sources
> Adding source map references
Uploading sourcemaps for release <MY_PACKAGE_NAME>.staging-1.1 distribution 2
> Uploading source maps for release <MY_PACKAGE_NAME>.staging-1.1

Source Map Upload Report
  Minified Scripts
    ~/index.android.bundle (sourcemap at index.android.bundle.map)
  Source Maps
    ~/index.android.bundle.map
```

**Comment by HazAT** [comment]
Hey, thank you for reporting this.
It's basically a dup of https://github.com/getsentry/react-native-sentry/issues/234 though this issue is more specific about it :)
I am on it 👍

**Comment by fabriziomoscon** [comment]
Thanks @HazAT !
My goal is not to disable the submission of sourcemaps tough. I just want them to be submitted once. And I am using sentry.io not my own server.
If you want I can close this and move my comment on #234

**Comment by HazAT** [comment]
Hey, I've just released `0.25.0`, the error should be gone now :)

**Comment by fabriziomoscon** [comment]
👍 
great now the build process only process sourcemaps for the active flavor!

Thanks a lot
